PocketKit/Sources/PocketKit/ItemAction.swift

```diff
@@ -57,6 +57,16 @@ extension ItemAction {
         )
     }
 
+    static func reAdd(_ handler: @escaping (Any?) -> ()) -> ItemAction {
+        return ItemAction(
+            title: "Re-add",
```

Comment:
Quick thought: maybe we shouldn't even use "Archive" / "Re-add" as titles, but "Move to My List" and "Move to Archive" and be more specific. Definitely a product question. It adds length, but also adds clarity imo.

Reply:
Yeah, I agree that the copy here could use some more thought. I went with "Re-add" per [recommendation](https://range.co/getpocket/checkins/AB8BFtGRHvac2_ZNDAAy?utm_medium=integration&utm_source=slack) from @marcin-kozinski but we can discuss with the team and tweak if necessary.